pkg/dao/types/connector.go

```diff
@@ -5,6 +5,8 @@ const (
 	ConnectorTypeKubernetes string = "Kubernetes"
 	ConnectorTypeAlibaba    string = "Alibaba"
 	ConnectorTypeAWS        string = "AWS"
+	ConnectorTypeAzureRM    string = "AzureRM"
```

Comment:
This is because the name of the provider is azurerm. Besides azurerm, Azure has another provider called azurerd, which is used for user and identity management. The UI will show AzureRM to avoid confusion.

Reply:
Simply show Azure in the UI. Similar to the following
<img width="1225" alt="image" src="https://github.com/seal-io/walrus/assets/5697937/c2d4ab16-bb3f-4efd-9c93-077d96be5db3">

Reply:
cc @hibig

Reply:
Updated. Changed connector type to Azure